participants/fmusic for slingshot.md

```diff
@@ -0,0 +1,60 @@
+###(0) Project Name and Team Slack Handles
+- Our project Name： FMusic
+- Team Slack Handles：[FUNBLOCKS](https://funbloks.slack.com/join/shared_invite/zt-hy0c64sg-pGNuL92B~azTpyZz8M6jgg#/)
+
+###(1) Brief Project Description
+- FMusic is a global  music source shared system. 
+- Users can upload the music they like and the music they want to collect.
```

Comment:
How do you protect that the user is not uploading data that they don't have licensed or isn't part of the free music archive?